README.md

```diff
@@ -41,14 +41,70 @@ drm= {{
     type: 'fairplay',
     licenseServer:'https://drm-fairplay-licensing.axtest.net/AcquireLicense',
```

Comment:
Align this part also with changes made in code:
1. In code you know use axprod URL. Use it here as well.
2. In code you've removed "chucked" transfer-encoding ... remove here as well.
3. Check if anything else like this.

Reply:
I will do it.